pkg/spanfilter/spanfilter.go

```diff
@@ -56,17 +56,21 @@ func (f *SpanFilter) ApplyFilterPolicy(rs *v1.Resource, span *tracev1.Span) bool
 		return true
 	}
 
+	var policyMatch int
+
 	for _, policy := range f.filterPolicies {
 		if policy.Include != nil && !policy.Include.Match(rs, span) {
-			return false
+			continue
 		}
 
 		if policy.Exclude != nil && policy.Exclude.Match(rs, span) {
-			return false
+			continue
 		}
+
+		policyMatch++
```

Comment:
can't we just return true here instead of incrementing policyMatch and evaluating other policies?

Reply:
When testing this it showed me that we needed to also handle the case where only have exclude policies are in place.  I've adjusted the logic to be a bit more verbose, and added additional tests and docs.